README.md

```diff
@@ -10,6 +10,25 @@ https://www.verdaccio.org
 
 Documentation files are hosted under `/docs` folder.
 
+# Running the website locally
+You should run all these tasks from the inner `/website` folder.
+
+The commands you need to use for serving the site locally:
+- `npm install` - installs all dependencies.
+- `npm start` - runs a local server that launches the Verdaccio documentation site on http://localhost:3000/
+
+# Running the website as a docker container
+
+The Verdaccio documentation website can run as a docker container (useful for offline stations, local serve of docs, etc.).
```

Comment:
stations => instsllations / usage
local serve? The wording could be a bit better.

Reply:
Text changed, hope it's ok now :)